File: backend-api/src/DTOs/OrdersCreatePayNowDTO.cs
namespace unipos_basic_backend.src.DTOs
{
    public sealed class OrdersCreatePayNowDTO
    {
        public Guid CashRegisterId { get; set; }
        public PymtMethodDTO? Method { get; set; }
        public List<OrderItemsDTO>? OrderItems { get; set; }
    }

    public sealed class PymtMethodDTO
    {
        public decimal? Cash { get; set; }
        public decimal? EMola { get; set; }
        public decimal? MPesa { get; set; }
    }
}

File: frontend/src/app/_interfaces/orders-create-pay-now-dto.ts
export interface OrdersCreatePayNowDTO {
  "cashRegisterId": string
  "method": PymtMethodDTO,
  "orderItems": {
    "productId": string
    "quantity": number
  }[]
}

export interface PymtMethodDTO {
  "cash": number
  "eMola": number
  "mPesa": number
}
